plugins/feeds/public/sslblacklist_ja3.py

```diff
@@ -0,0 +1,57 @@
+from datetime import timedelta
+from io import StringIO
+from typing import ClassVar
+
+import pandas as pd
+
+from core import taskmanager
+from core.schemas import task
+from core.schemas.observables import ja3
+
+
+class SSLBlacklist_JA3(task.FeedTask):
```

Comment:
Let's remove the `_` here to stay a bit consistent with other Feed classes.

```suggestion
class SSLBlacklistJA3(task.FeedTask):
```